gpt4all-chat/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Change the error message when a message is too long ([#3004](https://github.com/nomic-ai/gpt4all/pull/3004))
 - Simplify chatmodel to get rid of unnecessary field and bump chat version ([#3016](https://github.com/nomic-ai/gpt4all/pull/3016))
 - Allow ChatLLM to have direct access to ChatModel for restoring state from text ([#3018](https://github.com/nomic-ai/gpt4all/pull/3018))
+- Make it possible to downgrade and keep some chats ([#3030](https://github.com/nomic-ai/gpt4all/pull/3030))
```

Comment:
```suggestion
- Make it possible to downgrade and keep some chats while preventing crashes for others ([#3030](https://github.com/nomic-ai/gpt4all/pull/3030))
```

Reply:
Frankly, I wonder if this is a feature or should be mentioned in 'Fixed' instead as it does prevent crashes.

Reply:
Changed.